tests/test_utils.py

```diff
@@ -25,7 +23,13 @@ def port(pytestconfig):
 
 
 @pytest.fixture
-def instance(port):
+def instance(port, pytestconfig):
+    # The None record mode is the default and is equivalent to "once"
+    if pytestconfig.getoption("record_mode") in ["none", "once", None]:
```

Comment:
Should this be `getoption("--record-mode")`

Reply:
No, this gives us an argparse namespace that removes the leading `--`.